lib/schedule.js

```diff
@@ -619,6 +615,10 @@ function scheduleJob() {
   const method = name ? arguments[2] : arguments[1];
   const callback = name ? arguments[3] : arguments[2];
 
+  if (typeof method !== 'function') {
+    return null;
```

Comment:
Why return quietly instead of throwing an error? Wouldn't that result in users logic silently stopping from working?

Reply:
I just followed the pattern from [here](https://github.com/node-schedule/node-schedule/pull/557/files/eefe1d9c975cafcec798b5d5f20ac30524e4065b#diff-0389d465cec6cc3b4490de4118efa93f518aadd40a83a5df1c7190997d8d25ecR610), which returns `null` if the minimum required arguments are not passed. I agree it would make more sense to throw an error in both cases. I think there might be an issue about it. Should I change both?

Reply:
Yes, please!